extensions/amazon-aws/src/s3.tsx

```diff
@@ -54,7 +57,22 @@ function S3Bucket({ bucket }: { bucket: Bucket }) {
 }
 
 function S3BucketObjects({ bucket, prefix = "" }: { bucket: Bucket; prefix?: string }) {
-  const { data: objects, error, isLoading } = useCachedPromise(fetchBucketObjects, [bucket.Name || "", prefix]);
+  const [isReversedOrder, setReversedOrder] = useCachedState<boolean>("reverse-order", false, {
+    cacheNamespace: "aws-s3",
+  });
```

Comment:
Suggestion: Since `S3BucketObjects` gets called recursively (in case of folders), it'd be better if we specified this cached state at a global level here:

https://github.com/raycast/extensions/blob/462d9d961c15a4add84d9bb4d5ebb65188dec5ea/extensions/amazon-aws/src/s3.tsx#L20

and then pass `isReversedOrder` and `setReversedOrder` around through props.

Reply:
done

Reply:
This messed things up for some reason, please revert this change and resolve this. See the other thread for explanation.